.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,86 @@
+name: Docker Image CI
+
+on:
+  push:
+    tags:
+      - '*' 
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME_DAPP: ${{ github.repository }}-dapp
+  IMAGE_NAME_PUBLISHER: ${{ github.repository }}-publisher
+
+jobs:
+  build-and-push-dapp:
+    runs-on: ubuntu-latest
```

Comment:
Any particular reason for ubuntu?

Reply:
Kind of recommended by Github in all of their examples. No particular reason, we can use alpine as well. https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages

Reply:
Understood. If it would not require any big changes, maybe let's switch?

Here's a nice comparison: https://gist.github.com/thaJeztah/2071d4ddd50037a13646aa0f86089f96

Reply:
Sure, on it

Reply:
It looks to me Github doesn't provide alpine venv runners. Lets keep it as is for now.